keras_nlp/layers/start_end_packer.py

```diff
@@ -26,9 +26,10 @@ class StartEndPacker(keras.layers.Layer):
     be called after tokenization. The layer will first trim inputs to fit, then
     add start/end tokens, and finally pad, if necessary, to `sequence_length`.
 
-    If input is batched, input should be a `tf.RaggedTensor` with shape
-    `[batch_size, None]` and will be packed and converted to a dense tensor with
-    shape `[batch_size, sequence_length]`.
+    If input is batched, input should be either a `tf.RaggedTensor` with shape
```

Comment:
I think we can probably simplify this, and just say inputs can be either rank 1 or 2, and ragged or dense.